src/context.rs

```diff
@@ -2,10 +2,13 @@ use std::collections::HashMap;
 use value::Value;
 use filters::Filter;
 
+type ValueMap = HashMap<String, Value>;
+
 #[derive(Default)]
 pub struct Context {
-    values: HashMap<String, Value>,
-    pub filters: HashMap<String, Box<Filter>>,
```

Comment:
I'm good with the convenience that `add_filter` offers, but is it really necessary to break backwards compatibility by making `filters` private?